### PR TITLE
CONTRAfold-2.02-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**
added required patch to compile with gcc-4.7.2

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/83c07826394ce8145956 for a full test report.

**Comment by rjeschmi** [comment]
jenkins: test this please

File: easybuild/easyconfigs/c/CONTRAfold/CONTRAfold-2.02-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+name = 'CONTRAfold'
+version = '2.02'
+
+easyblock = 'MakeCp'
```
Comment (rjeschmi) [inline easybuild/easyconfigs/c/CONTRAfold/CONTRAfold-2.02-goolf-1.4.10.eb]: should this easyblock def be at the top?

Comment (boegel) [inline easybuild/easyconfigs/c/CONTRAfold/CONTRAfold-2.02-goolf-1.4.10.eb]: +1

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f27e5a80906d6c0d8e13 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks @pescobar !

**Pull request merged**